test/integration/publish_int_test.go

```diff
@@ -24,7 +27,7 @@ func (suite *PublishIntegrationTestSuite) TestPublish() {
 	suite.OnlyRunForTags(tagsuite.Publish)
 
 	// For development convenience, should not be committed without commenting out..
-	// os.Setenv(constants.APIHostEnvVarName, "staging.activestate.build")
+	// os.Setenv(constants.APIHostEnvVarName, "pr11496.activestate.build")
```

Comment:
Do you still want to keep this?

Reply:
Yeah just for convenience for now. We'll likely be able to drop this once we have a testing strategy for staging.